clox/params.py

```diff
@@ -23,6 +23,7 @@
 HORIZONTAL_TIME_12H_FORMATS = ['%I:%M %p', '%I:%M %p.']
 VERTICAL_TIME_12H_FORMATS = ['%I\n%M\n%p', '%I\n%M\n%p.']
 DATE_FORMAT = "%A, %B %d, %Y"
+TIMEZONE_DIFFERENCE_FORMAT = "{hours:02}h{minutes:02}m {direction}"
```

Comment:
I'm not sure about `ahead` and `behind`. The `+/-` signs are more intuitive.

Reply:
fixed in https://github.com/sepandhaghighi/clox/pull/49/commits/8c1c4a3e8ff5d16e11dcdaf3c5e78cb40f4433d7 and https://github.com/sepandhaghighi/clox/pull/49/commits/a68ab8896c2d8e2976c865b83de11f41ce966aae.

Now it's like the format that GitHub uses to display timezone differences.

![image](https://github.com/user-attachments/assets/885b822e-edf4-4f0e-a728-4189873377df)

Reply:
I guess that `-` is not the sign, that's just a dash separating (for example):
<img width="296" alt="Screenshot 2025-04-07 at 8 59 12 AM" src="https://github.com/user-attachments/assets/5c75d9e7-0e9d-44c9-af00-284d788d0c17" />